movie-browser/src/App.js

```diff
@@ -1,18 +1,30 @@
-import React from 'react';
+import React, { Component } from 'react';
 import Header from './components/Header.js';
 import Featured from './components/Featured.js';
 import Movies from './components/Movies.js';
 import Footer from './components/Footer.js';
+//import fetchMovies from './components/Api.js';
```

Comment:
delete comments. Same for line 16

Reply:
I left them in there for the future use of api.js but wanted the app to work before refactoring. I can remove if you still think it should be.

Reply:
oh, that's ok then. just leave them for now.